contentcuration/contentcuration/decorators.py

```diff
@@ -30,7 +31,7 @@ def wrap(request, *args, **kwargs):
         if request.user.is_admin:
             return function(request, *args, **kwargs)
 
-        return render_to_response('unauthorized.html', context_instance=RequestContext(request), status=403)
+        return render(request, 'unauthorized.html', status=403)
```

Comment:
the `context_instance` parameter of the `render_to_response` shortcut was deprecated in Django 1.10